cli/cli.go

```diff
@@ -16,5 +16,11 @@ func GetJfrogCliSecurityApp() components.App {
 		Commands:    getXrayNameSpaceCommands(),
 		Category:    "Command Namespaces",
 	})
+	app.Subcommands = append(app.Subcommands, components.Namespace{
+		Name:        string(cliutils.Sbom),
+		Description: "Sbom commands.",
+		Commands:    getSbomCommands(),
+		Category:    "sbom Namespaces",
```

Comment:
```suggestion
		Category:    "Command Namespaces",
```
This is the category of the Namespce to group namespaces together, it needs to have this value.

Reply:
ok